ptxconftools/confcontroller.py

```diff
@@ -1,6 +1,7 @@
 import sys
 import subprocess
 import re
+import numpy as np
```

Comment:
Is Numpy really used here?

Reply:
Thanks @huynle for working on this code... I and Wen-Hsin have not touched it unfortunately for years.
If you don't mind I'll add you as maintainer.
I don't recall if numpy was used anywhere in fact, so remove if see it anywhere, and ideally this whole repo needs cleaning up with `black` formatting and `flake8` linting.
```
$ flake8 ./
```
gives me a few other unused imports :p

Reply:
Also, because python3 now, best to change `# /usr/bin/python` to `#/usr/bin/python3` at top of `ptxconf.py`

Reply:
definitely, i'll get this cleaned up and documented. Numpy was accidentally left in there because i was playing with the coordinate transformation, which is no longer needed

Reply:
You should have received invitation from wen and can now merge it yourself I hope.
It may be a good idea to create a special space for the project and transfer it instead of keeping under /wenhsinjen/ personal account - github will create a redirect when projects are transferred.